### PR TITLE
update docs with new architecture

### PR DESCRIPTION
**Pull request by heitorPB**
charms use configless slurm and we don't have slurm-configurator
anymore.

### CODE REVIEW

File: architecture.rst [hunk 1]
```diff
@@ -3,11 +3,12 @@
...
 OSD Architecture
 ================
-The Omnivector Slurm Distribution is built on a suite of automations called "charms".
-Charms are the operational components that describe the lifecycle of a SLURM cluster.
-A full deployment comes in the form of multiple charms, one for each component of
-SLURM. A "bundle" is a yaml file where multiple charms can be defined. We use bundles
-to describe the interconnectivity and configuration of groups of charms. 
+The Omnivector Slurm Distribution is built on a suite of automations called
+"charms".  Charms are the operational components that describe the lifecycle of
+a SLURM cluster.  A full deployment comes in the form of multiple charms, one
```
Comment (jamesbeedy) [inline architecture.rst:8]:
```suggestion
a SLURM cluster.  A full SLURM deployment comes in the form of multiple charms, one
```

File: architecture.rst [hunk 2]
```diff
@@ -3,11 +3,12 @@
 ================
 OSD Architecture
 ================
-The Omnivector Slurm Distribution is built on a suite of automations called "charms".
-Charms are the operational components that describe the lifecycle of a SLURM cluster.
-A full deployment comes in the form of multiple charms, one for each component of
-SLURM. A "bundle" is a yaml file where multiple charms can be defined. We use bundles
-to describe the interconnectivity and configuration of groups of charms. 
+The Omnivector Slurm Distribution is built on a suite of automations called
```
Comment (jamesbeedy) [inline architecture.rst:6]:
```suggestion
The Omnivector SLURM Distribution is built on a suite of automations called
```

File: architecture.rst [hunk 3]
```diff
@@ -18,7 +19,7 @@ needed to facilitate the lifecycle of a SLURM cluster.
 Slurm Bundles
 -------------
 The `slurm-bundles <https://github.com/omnivector-solutions/slurm-bundles/>`_
-define the base slurm deployment configurations for different clouds and
+define the base Slurm deployment configurations for different clouds and
```
Comment (jamesbeedy) [inline architecture.rst:22]:
```suggestion
define the base SLURM deployment configurations for different clouds and
```

File: operations/partitions.rst [hunk 4]
```diff
@@ -70,18 +70,18 @@ Alternatively, you can set the partition name in the bundle file:
 Setting the default partition
 #############################
 
-To set the default Slurm partition, you must configure ``slurm-configurator``:
+To set the default Slurm partition, you must configure ``slurmctld``:
```
Comment (jamesbeedy) [inline operations/partitions.rst:73]:
```suggestion
To set the default partition, you must configure ``slurmctld``:
```

**Comment by heitorPB** [comment]
Should we always write `SLURM` when referring to the Schedmd project, and `Slurm` when referring to `Omnivector Slurm Distribution`?
@jamesbeedy

**Pull request merged**